### PR TITLE
Feature Pusher

### PR DESCRIPTION
**Pull request by renanbona**
https://central.cm42.io/projects/central#story-20255

Added Pusher to help story's change
Pusher configured and tested with the keys from my account on pusher.com
The Board only updates when the pusher is trigged
The pusher is trigged when you create, update or delete one story

### CODE REVIEW

File: spec/operations/story_operations/pusher_notification_spec.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      @model = model
+      @user = user
+    end
+  end
+
+  let(:story_params) do
+    { title: 'Test Story', requested_by: user, state: 'unstarted', accepted_at: nil }
+  end
+
+  let!(:membership) { create(:membership) }
```
Comment (adbatista) [inline spec/operations/story_operations/pusher_notification_spec.rb]: you're not using this, if the record i required add in a before block, if is not just remove it

File: spec/operations/story_operations_spec.rb [hunk 2]
```diff
@@ -366,6 +384,22 @@
     end
   end
 
+  describe "::Destroy" do
+    before do
+      story.save!
+    end
+
+    subject { -> { StoryOperations::Destroy.call(story, user) } }
```
Comment (adbatista) [inline spec/operations/story_operations_spec.rb]: why are you wrapping in a block?

File: app/assets/javascripts/central.js [hunk 3]
```diff
@@ -7,6 +7,13 @@ var ProjectSearchView = require('views/project_search_view');
 var ProjectVelocityView = require('views/project_velocity_view');
 
 import TourController from 'controllers/tour/TourController';
+import Pusher from 'pusher-js';
+
+var pusherApiKey = process.env.PUSHER_APP_KEY;
```
Comment (adbatista) [inline app/assets/javascripts/central.js]: does `process.env` exposes all ENV variables?

Comment (talyssonoc) [reply app/assets/javascripts/central.js]: Only the ones you use. It replaces the code for each env var similarly to what is done by C's `#define`.

File: app/assets/javascripts/central.js [hunk 4]
```diff
@@ -7,6 +7,13 @@ var ProjectSearchView = require('views/project_search_view');
...
+var pusherCluster = process.env.PUSHER_APP_CLUSTER;
+var boardSocket = new Pusher(pusherApiKey,{
```
Comment (talyssonoc) [inline app/assets/javascripts/central.js]: What do you think about extracting the instantiation of Pusher to a separate file? Then you would only need to import the socket and use it. Something like:

```js
import { boardSocket } from './pusherSockets';
```

**Comment by andrerpbts** [comment]
Don't forget CHANGELOG

**Pull request merged**